package.json

```diff
@@ -11,10 +11,13 @@
     "lint": "npm run lint:js"
   },
   "dependencies": {
+    "@mdi/font": "^6.5.95",
     "@nuxtjs/axios": "^5.13.1",
     "@nuxtjs/pwa": "^3.3.5",
     "core-js": "^3.9.1",
-    "nuxt": "^2.15.3"
+    "nuxt": "^2.15.3",
+    "swiper": "^5.4.5",
```

Comment:
swiper seems to be included by vue-awesome-swiper already, so you don't have to include it here